### DISCUSSION TITLE
@Clerk/Nextjs 5.0.12: redirect query param dissapearing in factor-one step

### DISCUSSION THREAD

**Discussion by andresvial** [clerk/javascript#3403] [Support request - DO NOT OPEN HERE] [answered]
When using `signInFallbackRedirectUrl` in the `redirectToSignIn` method, it redirects me to the sign-in page, with the redirect as a query param, which is expected. 

The problem occurs when I input the email and the press continue to input the password, where the redirect param dissappears and the urls ends up only being `sign-in/factor-one`, without the query params to redirect. This makes so when I sign in, I don't get redirected to the page I was originally trying to access.

Is this a bug or am I doing something wrong

**Comment by desiprisg** [comment]
Hey there! Thanks for reporting this. We've identified the issue and are actively working to resolve. Thanks!

**Comment by desiprisg** [comment, marked as answer]
This should be fixed in `@clerk/nextjs@5.1.1`.

**Reply by artiefy** [reply]
It happened to me in version 6, the redirection does not work because it captures the url but it should redirect me to the previous url that it captured it always sends me to / 

http://localhost:3000/sign-in#/?redirect_url=http%3A%2F%2Flocalhost%3A3000%2Fplanes

It should redirect me to /plans but it told me sending to / ?? please fixed
